app/controllers/solidus_klarna_payments/callbacks_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module SolidusKlarnaPayments
+  class CallbacksController < DeprecationController
```

Comment:
Inheritance makes a strict relation with the parent, what do you think to make a controller helper so you just have to include it?

https://github.com/solidusio/solidus/blob/d996288936e7bc60359fedaaa87aba5d8ff01cf1/core/lib/spree/core/controller_helpers/current_host.rb#L6